includes/class-notices.php

```diff
@@ -631,4 +631,48 @@ public static function print_frontend_notice( $message, $echo = true, $notice_ty
 
 		echo $error;
 	}
+
+	/**
+	 * Print Inline Notice.
+	 *
+	 * @param string $message     Message Text.
+	 * @param array  $notice_args An array of notice arguments.
+	 *
+	 * @todo Implement render_admin_notices function within this function in future.
+	 *
+	 * @access public
+	 * @since  1.8.17
+	 *
+	 * @return string
+	 */
+	public function print_admin_notices( $message, $notice_args = array() ) {
```

Comment:
@mehul0810 Add PHPUnit test for this function

Reply:
@ravinderk Added Unit Tests